bundles/org.openhab.binding.rotel/src/main/resources/OH-INF/i18n/rotel.properties

```diff
@@ -110,6 +110,14 @@ channel-type.rotel.treble.label = Treble Adjustment
 channel-type.rotel.treble.description = Adjust the treble
 channel-type.rotel.volumeUpDown.label = Volume
 channel-type.rotel.volumeUpDown.description = Increase or decrease the volume
+channel-type.rotel.bypassx.label = Bypass Adjustment
+channel-type.rotel.bypassx.description = Switch Bypass On/Off
+channel-type.rotel.balance.label = Balance Adjustment
+channel-type.rotel.balance.description = Adjust the Balance Left-Right
+channel-type.rotel.speakera.label = Speaker-A Adjustment
+channel-type.rotel.speakera.description = Switch Speaker-A On/Off
+channel-type.rotel.speakerb.label = Speaker-B Adjustment
+channel-type.rotel.speakerb.description = Switch Speaker-B On/Off
```

Comment:
As a start point, please have same values as in the XML file. 
Decide which are the best.

Reply:
ok: I made requested adjustments